The-SMIRNOFF-force-field-format.md

```diff
@@ -1,103 +1,288 @@
+<vdWForce potential="Lennard-Jones" scale12="0.0" scale13="0.0" scale14="0.5" scale15="1" sigma_unit="angstroms" epsilon_unit="kilocalories_per_mole">
    <Atom smirks="[#1:1]" rmin_half="1.4870" epsilon="0.0157"/>
    <Atom smirks="[#1:1]-[#6]" rmin_half="1.4870" epsilon="0.0157"/>
    ...
-</NonbondedForce>
+</vdWForce>
+```
+Attributes in the `<vdWForce>` tag specify the scaling terms applied to the energies of 1-2 (`scale12`, default: 0), 1-3 (`scale13`, default: 0), and 1-4 (`scale14`, default: 0.5) interactions, (`scale15`. default: 1.0).
+
+Currently, only `potential="Lennard-Jones-12-6"` is supported:
+```
+U(r) = 4*epsilon*((sigma/r)^12 - (sigma/r)^6)
+```
+Later revisions will add support for additional potential types (e.g., `Buckingham-exp-6`) and the ability to support arbitrary algebraic functional forms.
+If the `potential` attribute is omitted, it defaults to `Lennard-Jones-12-6`.
+
+**QUESTION:** We need to specify mixing rules as well. How should we do this? Via a single attribute like `mixing_rules="Lorentz-Berthelot"`, via multiple attributes like `sigma_mixing_rule="geometric" epsilon_mixing_rule="arithmetic"`, or via a block like
+```XML
+<vdWForce energy_expression="4*epsilon*((sigma/r)^12-(sigma/r)^6)" scale12="0.0" scale13="0.0" scale14="0.5" sigma_unit="angstroms" epsilon_unit="kilocalories_per_mole">
+   <CombiningRules>
+      <CombiningRule parameter="sigma" function="(sigma1+sigma2)/2"/>
+      <CombiningRule parameter="epsilon" function="sqrt(epsilon1*epsilon2)"/>
+   </CombiningRules>
+   ...
+</vdWForce>
+```
+Other [possibilities](https://en.wikipedia.org/wiki/Combining_rules) we can support later: `Waldman-Hagler`, `Fender-Halsey`, `Kong`, `Tang-Toennies`, `Pena`, `Hudson-McCoubrey`, `Sikora`.
+
+**QUESTION:** How would we specify an LJ interaction table among SMIRKS-specified types? Perhaps via an `<AtomPair/>` tag?
+```XML
+<vdWForce potential="Lennard-Jones-12-6" scale12="0.0" scale13="0.0" scale14="0.5" sigma_unit="angstroms" epsilon_unit="kilocalories_per_mole">
+   <AtomPair smirks1="[#1:1]" smirks2="[#6:2]" sigma="1.4870" epsilon="0.0157"/>   
+   ...
+</vdWForce>   
 ```
-Scaling terms for 1-4 interactions should be specified in attributes for the `NonbondedForce` tag, along with units.
 
-For compatibility, the size property of an atom can be specified either via providing the `sigma` attribute, such as `sigma="1.3"`, or via the `r_0/2` (`rmin/2`) values used in AMBER force fields (here denoted `rmin_half` as in the example above). The two are related by `r0 = 2^(1/6)*sigma` and conversion is done internally in `ForceField` into the `sigma` values used in OpenMM. `epsilon` denotes the well depth.
+**QUESTION:** Should we include cutoff, switch, and long-range correction flags, or whether LJPME is used? For LJ, there are ways to select these so that physical and binding properties are less dependent on these properties, but this may not be true of other kinds of potentials.
```

Comment:
I'm in favor of cutoff, switch, and long range correction. Less so for LJPME since it's not necessarily common everywhere yet.

Reply:
I would include cutoff, switch, long range correction and LJPME. Include the latter because if it was part of the parametrization, we need to request it. Using the parameters in an engine which does not have LJPME should fail or throw a warning in such a case.